setup.bat

```diff
@@ -1,3 +1,3 @@
 setlocal
 
-rustup toolchain install nightly --component rust-src rustc-dev llvm-tools-preview
+rustup toolchain install nightly-2020-10-24 --component rust-src rustc-dev llvm-tools-preview
```

Comment:
Wouldn't `rustup component add rust-src rustc-dev llvm-tools-preview` automatically install the correct nightly?

Reply:
Probably, if it does read the `rust-toolchain` file, worth trying